pandas/core/arrays/arrow/array.py

```diff
@@ -1612,15 +1612,20 @@ def to_numpy(
             result[~mask] = data[~mask]._pa_array.to_numpy()
         return result
 
-    def map(self, mapper, na_action: Literal["ignore"] | None = None):
+    def map(self, mapper,
+            na_action: Literal["ignore"] | None = None,
+            preserve_dtype: bool = False):
```

Comment:
why is a new keyword getting added?

Reply:
i see, you're trying to avoid changing any old behavior.  in this case, the new behavior should be generally better than the old behavior.  if you find cases where it isn't, those likely represent bugs in _cast_pointwise_result.